docfx.json

```diff
@@ -109,6 +109,11 @@
       "apiPlatform": "dotnet",
       "ms.prod":"dotnet",
       "ms.topic": "conceptual",
+      "_op_documentIdPathDepotMapping": {
+	"docs/standard/": { 
```

Comment:
should we do this for each e-book folder instead? The standard folder is staying, just 5 folders are moving,